.github/workflows/ci.yml

```diff
@@ -16,19 +16,36 @@ jobs:
       - name: Noop
         run: "true"
   ci:
-    name: (${{ matrix.os }}) CI
+    name: (${{ matrix.name }}) CI
     needs: org-check
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # N.B.: macos-13 is the oldest non-deprecated Intel Mac runner and macos-14 is the oldest
         # non-deprecated ARM Mac runner.
-        os: [ ubuntu-22.04, linux-arm64, macos-13, macos-14, windows-2022, windows-arm64 ]
+        include:
+          - os: ubuntu-24.04
+            name: Linux x86-64
+            docker-platform: linux/amd64
+          - os: ubuntu-24.04
+            name: Linux aarch64
+            docker-platform: linux/arm64
```

Comment:
@jsirois I changed the Linux x86-64 and aarch64 build to use Docker as well, so it is consistent between this repo and pex. Let me know if you want me to change it back.

Reply:
That's a good change. The Linux arm and Windows arm machines are my personal machines and it's good to get one of them out of the loop.

Reply:
Ahh okay. I thought you had access to the GitHub arm runners. I got access to them at work, as we got a paid plan :)